nconf/args.go

```diff
@@ -0,0 +1,71 @@
+package nconf
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/pflag"
+)
+
+type RootArgs struct {
+	Prefix  string
+	EnvFile string
+}
+
+func (args *RootArgs) Setup(config interface{}, version string) logrus.FieldLogger {
```

Comment:
I am not a huge fan of a ton of packages, especially when there are only a few functions in that package. I think it can cause a lot of fragmentation. 

I think that as we get some more standard stuff (e.g. metrics, dist tracing) we can tackle that then

Reply:
I would also consider bringing something like this back: 
https://github.com/netlify/netlify-commons/tree/4c5e85925d783a4957766c34433d6352a55634bc/metrics